spring-cloud-gcp-data-spanner/src/main/java/com/google/cloud/spring/data/spanner/core/convert/ConverterAwareMappingSpannerEntityWriter.java

```diff
@@ -361,14 +368,23 @@ private static <T> boolean attemptSetSingleItemValue(
     return true;
   }
 
-  private Value covertJsonToValue(Object value) {
+  private Value convertJsonToValue(Object value) {
     if (value == null) {
       return Value.json(null);
     }
     String jsonString = this.spannerMappingContext.getGson().toJson(value);
     return Value.json(jsonString);
   }
 
+  private Iterable<String> covertIterableJsonToValue(Iterable<Object> value) {
```

Comment:
You fixed the covert conversion typo but not before copying it over :)

Reply:
ah, good catch!!